website/views/shared/tasks/lists.jade

```diff
@@ -27,7 +27,9 @@ script(id='templates/habitrpg-tasks.html', type="text/ng-template")
             textarea(rows='6', task-focus='list.bulk && list.focus', ng-model='list.newTask', placeholder='{{list.placeHolderBulk}}', ng-if='list.bulk', ui-keydown='{"meta-enter ctrl-enter":"addTask(obj[list.type+\'s\'],list)"}', required)
             input(type='text', task-focus='!list.bulk && list.focus', ng-model='list.newTask', placeholder='{{list.placeHolder}}', ng-if='!list.bulk', required)
             button(type='submit', ng-disabled='new{{list.type}}form.$invalid')
-              span.glyphicon.glyphicon-plus
+             div.empty-task-notification( ng-show='new{{list.type}}form.$invalid', tooltip=env.t("emptyTask") )
```

Comment:
I think the tooltip should be attached to the button above this, and it should just have conditional text, like this:

```
button(type='submit', ng-disabled='new{{list.type}}form.$invalid', tooltip={{list.newTask == '' ? env.t("emptyTask")) : ''}})
```

Again, doing this from memory, so I may have syntax errors. If it works, this should remove the need for the extra class and surrounding div. It should also position the tooltip better, so it doesn't overlap on the button as much.